workflow-beta2/tpl/deis-logger-fluentd-daemon.yaml

```diff
@@ -0,0 +1,45 @@
+#helm:generate helm tpl -d $HELM_GENERATE_DIR/tpl/generate_params.toml -o $HELM_GENERATE_DIR/manifest/deis-logger-fluentd-daemon.yaml $HELM_GENERATE_DIR/tpl/deis-logger-fluentd-daemon.yaml
```

Comment:
good catch. fixed inhttps://github.com/deis/charts/pull/224/commits/88090d645a43c51f0e60c7acc72d9a5e275a1676 and restarting the [failed jenkins job](https://ci.deis.io/job/workflow-test-pr/709/console)